MaxText/configs/base.yml

```diff
@@ -63,15 +63,20 @@ int8_training: False
 # base_mlp_dim, base_num_decoder_layers and/or head_dim.
 global_parameter_scale: 1
 base_emb_dim: 2560
+base_num_heads: 8
 base_num_query_heads: 8
 base_num_kv_heads: 8
 base_mlp_dim: 8192
 base_num_decoder_layers: 16
 head_dim: 256
+
 # activation functions are .
 mlp_activations: ["relu"]
 dropout_rate: 0
 logits_via_embedding: True  # NOTE: this is True just for testing.
+norm_logits_via_embedding: True  # whether to normlize pre-softmax logits if logits_via_embedding is true
```

Comment:
The comment is much more descriptive than variable. Shouldn't it be 'normalize_embedding_logits'? (Obviously correct me if I misunderstand.

Reply:
This key is activated only with `logits_via_embedding=True`. Originally, I hope to show the info in naming the key and use the same postfix as `logits_via_embedding`
I think `normalize_embedding_logits` is more understandable, will change it.  

```
   if cfg.logits_via_embedding:
      # Use the transpose of embedding matrix for logit transform.
      logits = self.shared_embedding.attend(y)
      if self.config.norm_logits_via_embedding:
        # Correctly normalize pre-softmax logits for this shared case.
        logits = logits / jnp.sqrt(y.shape[-1])
    else:
      logits = linears.DenseGeneral(
          cfg.vocab_size,
          dtype=jnp.float32 if cfg.logits_dot_in_fp32 else cfg.dtype,  # for logit training stability
          kernel_axes=('embed', 'vocab'),
          name='logits_dense',
          use_int8=cfg.int8_training)(y)
```